### PR TITLE
http: correctly handle CORS parameter when creating THttpServer

### PR DESCRIPTION
**Pull request by linev**
It can be specified as extra option of THttpServer itself like:

   new THttpServer("http:8080;cors")
   
But also as in original code:   

   new THttpServer("http:8080?cors")
   
Second form is formally belongs to the TCivetweb engine and now handled
appropriately

### CODE REVIEW

File: net/http/inc/THttpServer.h [hunk 1]
```diff
@@ -44,7 +44,8 @@ class THttpServer : public TNamed {
    TString fDefaultPageCont; ///<! content of the file content
    TString fDrawPage;        ///<! file name for drawing of single element
    TString fDrawPageCont;    ///<! content of draw page
-   TString fCors;            ///<! CORS (cross-origin resource sharing): sets Access-Control-Allow-Origin for ProcessRequest responses
+   TString
```
Comment (bellenot) [inline net/http/inc/THttpServer.h]: This one looks weird to me...

Comment (linev) [reply net/http/inc/THttpServer.h]: It is result of clang-format. 
Probably, due to long comments

File: net/http/inc/THttpServer.h [hunk 2]
```diff
@@ -44,7 +44,8 @@ class THttpServer : public TNamed {
...
+      fCors; ///<! CORS (cross-origin resource sharing): sets Access-Control-Allow-Origin for ProcessRequest responses
```
Comment (bellenot) [inline net/http/inc/THttpServer.h]: This one looks weird to me...

Comment (bellenot) [reply net/http/inc/THttpServer.h]: OK, just ignore my comment...

Comment (linev) [reply net/http/inc/THttpServer.h]: If I will put it in one line, Travis CI will complain

**Pull request merged**